internal/runners/clean/run_lin_mac.go

```diff
@@ -105,6 +105,12 @@ func removeInstall(cfg configurable) error {
 		}
 	}
 
+	if transitionalStatePath := cfg.GetString(installation.CfgTransitionalStateToolPath); transitionalStatePath != "" {
+		if err := os.Remove(transitionalStatePath); err != nil && !errors.Is(err, os.ErrNotExist) {
+			aggErr = errs.Wrap(aggErr, "Could not remove %s: %v", transitionalStatePath, err)
+		}
+	}
```

Comment:
This is not what this story is about. We will do this, but only after enough time has passed. It'd be too soon to remove it.

This story is about removing it when you run `state clean uninstall`.

Reply:
Yes, this is called during `state clean uninstall`.

Reply:
Oh sorry, I missed that this was in a different file.